apps/entry/views.py

```diff
@@ -49,7 +52,41 @@
 import django_filters
 
 
-class EntryViewSet(viewsets.ModelViewSet):
+class EntrySummaryPaginationMixin(object):
+    def get_paginated_response(self, data):
+        if not self.request.data.get('calculate_summary',
+                                     self.request.GET.get('calculate_summary', '0')) == '1':
+            return super().get_paginated_response(data)
+        qs = self.filter_queryset(self.get_queryset())
+        q = qs.annotate(org=models.functions.Coalesce('lead__authors__parent', 'lead__authors')).values('org').annotate(org_type=models.functions.Coalesce('lead__authors__parent__organization_type', 'lead__authors__organization_type')).values('org_type').order_by('org_type').annotate(count=models.Count('org', distinct=True)).values('org_type', 'count')
+        org_type_count = []
+        for each in q:
+            if each.get('count') and each.get('org_type'):
+                org_type = OrganizationType.objects.get(id=each['org_type'])
+                org_type_count.append(
+                    dict(org=dict(
+                       id=each.get('org_type'),
+                       title=org_type.title,
+                       shortName=org_type.short_name,
+                    ),
+                    count=each.get('count'))
+                )
```

Comment:
I am not sure but you should be able to get this data directly from the queryset.
```suggestion
        org_type_count = list(
            q.values('count', id=F('org_type__id'), title=F('org_type__title'), short_name=F('ort_type__short_name'))
        )
```

Make sure to remove the last `.values(` from q declaration. Also, if it throws an error like **there is a column already**, then use list comprehension for defining dict manually.

Reply:
Also, try not to use camelCase in python. Specifed DRF renderer will do that for us (if `list(` is not used then DRF renderer may not convert the snake_case to CamelCase)